### DISCUSSION TITLE
#BravoBiniou

### DISCUSSION THREAD

**Discussion by koinkoin-project** [Woolverine94/biniou#7] [Show and tell]
Hello,

Je voulais partager avec toi ma toute première vidéo au format TikTok générée par une IA GAFAM. Voici le lien : [https://www.youtube.com/watch?v=Yj-Bht3qF_8](url)

Je n'ai absolumment aucune compétences youtube et de montage video, donc d'après moi tout ce qui peut etre fait par AI doit etre fait par AI . Je fais là mes premiers pas, encore :)

Ce que j'apprécie dans cette vidéo :

J'aime particulièrement la voix américaine utilisée.
Le rythme haletant de la voix est vraiment captivant.
La cadence globale de la vidéo est rapide et dynamique.
Les textes et les sous-titres apportent un contexte significatif.

Ce que je n'apprécie pas autant :

Cependant, les images et les vidéos utilisées ne me convainquent pas totalement, même si cela pourrait rester tel quel de manière permanente.

Et malheureusement, mes ressources financières ne me permettent pas d'acheter les vidéos générées sans filigranes (pour le moment).
J'ai créé une chaîne YouTube pour Biniou.

Voici le lien : [Chaîne YouTube Biniou, future opération RS #BravoBiniou](https://www.youtube.com/@BiniouAIGenerative).

Le descriptif actuel de la chaîne est assez générique et peu attrayant, mais je prévois de le réviser précisément à la fin de l'année.

N'hésite pas à me donner ton avis sur cette vidéo et la chaîne, et si tu as des suggestions ou des questions, je serais ravi d'en discuter davantage. De plus, ce serait super si tu pouvais passer sur la chaîne pour y laisser des commentaires si tu trouves des vidéos intéressantes plus tard. Ça serait génial !
Salut,

**Comment by Woolverine94** [comment]
Bonjour @koinkoin-project ,

Merci pour vos encouragements et cette vidéo promotionnelle spontanée.

Je n'ai pas d'avis particulier sur la vidéo en elle-même, qui « fait le job ». Je suis en revanche plus réservé sur le message véhiculé, qui même s’il reprend l’argumentaire du README.md, fait l’impasse sur les réserves et limitations du logiciel (l’installation macOS n’a, à ma connaissance, jamais été testée et reste purement théorique), ainsi que sur son aspect expérimental. J’ai peur qu’au final, cela génère surtout de la déception en survendant les capacités de biniou. 

Concernant le fait de participer à votre chaîne Youtube, je ne suis absolument pas dans une démarche de promotion actuellement : biniou n’est pas encore assez stable pour cela. Je préfère attendre pour présenter un produit qui fonctionne, que de faire la promotion d’un produit qui ne fonctionne pas, et ainsi le tuer dans l’œuf. Je me focalise **exclusivement** à l’heure actuelle sur le développement de fonctionnalités, les correctifs et les mises à jour de biniou. Je note toutefois l’adresse de votre page Youtube, et ne manquerai pas de la consulter ;)

En vous remerciant.

**Reply by koinkoin-project** [reply]
> fait l’impasse sur les réserves et limitations du logiciel (l’installation macOS n’a, à ma connaissance, jamais été testée et reste purement théorique), ainsi que sur son aspect expérimental.

Je n'ai aucun abonné, et n'ai fait aucune view à ma connaissance :). De plus je ne cherche pas l'audience à tout prix (surtout pas) et je n'envisage que quelques vidéos biniou sur cette châine. Je n'ai pas énormement d'ambitions pour cette chaîne (à cause de mon manque de compétences et que ce n'est pas mon coeur de métier ni même une passion en soi) :). Je peux changer d'avis demain bien sûr, mais aujourdhui je n'ai pas spécialement de stratégie hormis prouver visuellement qu'on peut faire des trucs sympas avec biniou sur une machine vraiment modeste (et que c'est peut etre là une des DERNIÈRES solutions que les pas-riches ont, pour utiliser des outils génératives AI. Je sais de quoi je parle :), moi qui tourne sur une machine biniou avec du hard vraiment faiblard... Pour les MAC quelqu'un passera par ta page projet un ce ces jours, mais fais gaffe à MAC et leurs nouvelles bibilo AI Opensource ce seront les seuls bientôt à proposer des machines Mac IA avec gavé de RAM combinée (concurrence frontale avec NVIDIA)

> J’ai peur qu’au final, cela génère surtout de la déception en survendant les capacités de biniou.

Non car si dans le futur tu envisages un biniou stable alors cette vidéo vieillira bien.

> Je n'ai pas d'avis particulier sur la vidéo en elle-même, qui « fait le job ».

J'étais desespéré il me fallait une sorte de « trailer » et je suis content que tu la valides d'une certaine manière malgrès toutes tes réserves enoncées plus haut (que j'aisseraie de corriger bientôt).

J'envisages maintenant un futur trailer qui ressemblera à un format Zapping dans un ecran tv, avec des youtubers frenchies qui expliquent avec leurs mots que les IA locales peuvent etre plus puissantes que des IA GAFAM parceque plus spécialisées, que les IA en local c mieux et tout et tout (korben, scripting, confidentialité) etc. J'ai dix youtubers qui racontent chacuns quelques secondes de IAGAFAM/vs/IALOCAL

> Concernant le fait de participer à votre chaîne Youtube, je ne suis absolument pas dans une démarche de promotion actuellement : biniou n’est pas encore assez stable pour cela. Je préfère attendre pour présenter un produit qui fonctionne, que de faire la promotion d’un produit qui ne fonctionne pas, et ainsi le tuer dans l’œuf. 

Je le savais déjà :) Mais tout bon projet mérite une communauté et tout le monde parle de pinokio, mais personne ne parle de biniou (pourquoiii) !!
LOL

@+

**Comment by Woolverine94** [comment]
@koinkoin-project 

Pouvez-vous surtout préciser dès que possible sur votre chaîne Youtube qu'il ne s'agit pas de la chaîne officielle du projet ? 

Merci par avance.

**Reply by koinkoin-project** [reply]
Ce projet de chaine youtube n'est pas la chaine officielle de biniou mais une chaine de soutien de mon projet qui est plus vaste que biniou (et orienté Outils GUI / Modeles Local AI + scripting). J'ai déjà mentionné cela sur le site de mon projet principal. Je vais retoucher le texte de la vidéo et la description de la chaine bravobiniou pour éviter la moindre confusion (en fin d'année (dans une dizaine de jours)).

Saches que je communiquerai toujours sur biniou comme un outil plutôt génial pour les créatifs, que plutôt "pas abouti" (qui est pas très vendeur ni pour quiconque ni pour une chaine youtube, ni même pour les premières vidéos de présentation de n'importe quel tool). De toute facons, personne ne verra mon visage, je ne parlerai de binou que lorsque j'évoquerai le GUI ou ses onglets, et je parlerai des modèles et de leurs subtilités quand des problemes surviendront lors des générations foirées (comme ce vaisseau spatial 2d, raté en 3d que j'attribue à mes mauvais réglages et ma mauvaise utilisation du modele, peut être même à des problèmes de manque de RAM!? et biniou n'y est pour rien là). Je suis developpeur et à ce titre et dans toutes les vidéos tutos de biniou sur la chaine de #bravobiniou, je saurai faire la part entre biniou/conf/modeles. Biniou est un excellent tool, c'est le premier qui m'a enfin fait kiffer l'ia générative en local chez moi hors gafam, personne ne le dit sur la toile, et pour d'autres raisons je me sens le devoir d'incarner au moins un temps ce petit bout de web qui pourra traiter de biniou en particulier, d'ia locales en général dans un format tutoriel non marqueteux (hormis les trailer, mais tous mes projets ou tous les projets des autres que je mettrai en avant dans mes comm auront tous leur(s) trailer(s)).

Je t'ai senti vexé au dernier message, j'espére maintenant t'avoir rassuré en certains points.

**Reply by koinkoin-project** [reply]
J'ai modifié le descriptif de la chaine bravobiniou en précisant que c une chaine NON OFFICIELLE qui traite de biniou. J'ai modifié également le descriptif de la vidéo trailer pour ne laisser place a aucune confusion possible.

Je vais modifier plus tard la vidéo trailer pour le préciser aussi (que biniou a un auteur, pour mac aussi, pour le coté expérimental aussi). Je t'informerai quand ce sera fait et republié corrigé.

**Comment by koinkoin-project** [comment]
Bonjour @Woolverine94 

J'ai supprimé la chaine dédiée de BravoBiniou. Il n'y a normallement plus aucune trace de cette chaine.

Je ferais certainement des vidéos tuto dédiées biniou sur ma propre chaine youtube, parceque je pense encore que biniou est un bon soft (expérimental) qui mérite d'être diffusé auprès du grand public.

Bonne année et bonne continuation pour ce projet.

Cordialement

**Reply by Woolverine94** [reply]
Merci. 

Une bonne année à vous aussi. 

Cdt,
